### PR TITLE
TRN-301: Content Structure.

### PR DESCRIPTION
**Pull request by hiyampatel**

### CODE REVIEW

File: config/field.field.node.news.body.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    - field.storage.node.body
+    - node.type.news
+  module:
+    - text
+id: node.news.body
+field_name: body
+entity_type: node
+bundle: news
+label: 'News body'
+description: ''
```
Comment (kkmitra) [inline config/field.field.node.news.body.yml]: Add description to the fields.

File: config/image.style.news_image_500_300_.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+uuid: 1df4e5d2-40a5-44e1-852e-f6d9bca285f9
+langcode: en
+status: true
+dependencies: {  }
+name: news_image_500_300_
```
Comment (kkmitra) [inline config/image.style.news_image_500_300_.yml]: Change the image style, there shouldn't be any extra `_` at the end of a machine name.

**Review by kkmitra — CHANGES_REQUESTED** [review]

File: config/image.style.news_image_500_300_.yml (hunk 2 above)
Comment (hiyampatel) [reply config/image.style.news_image_500_300_.yml]: Changed machine name

File: config/field.field.node.news.body.yml (hunk 1 above)
Comment (hiyampatel) [reply config/field.field.node.news.body.yml]: Added description

**Pull request merged**